ads/google/utils.js

```diff
@@ -170,3 +170,25 @@ function validateDimensions(width, height, widthCond, heightCond, errorBuilder)
   }
   return !badParams.length;
 }
+
+/**
+ * Calculates height of responsive matched content slot based on its width.
+ * This logic should be kept as close to possible to the logic inside
+ * adsbygoogle.js.
+ *
+ * @param {number} width
+ * @return {number}
+ */
+export function getMatchedContentResponsiveHeight(width) {
+  if (width >= 1200) {
+    return 600;
+  } else if (width >= 850) {
+    return Math.floor(width * 0.5);
+  } else if (width >= 550) {
+    return Math.floor(width * 0.6);
+  } else if (width >= 468) {
+    return Math.floor(width * 0.7);
+  } else {
+    return Math.floor(width * 3.44);
```

Comment:
Did you mean for this to match the old logic in which case the multiplier should be 3.4 instead of 3.44 and you should add 112?